### PR TITLE
Adds Signature Rendering to PDF

### PR DESCRIPTION
**Pull request by qwwqwwq**

### CODE REVIEW

File: winterfell/pdf-filler/src/test/java/gov/va/vetaffi/PDFStampingTest.java [hunk 1]
```diff
@@ -39,10 +48,30 @@ public void testStampPdfWithCheckAndText() throws Exception {
...
+        PDFStamping.stampPdf(pdfTemplate,
+                Lists.newArrayList(
+                        new PDFField("signature", SIGNATURE)
+                ),
+                Lists.newArrayList(
+                        new PDFFieldLocator("F[0].Page_1[0].SignatureOfClaimant_AuthorizedRepresentative[0]",
+                                "signature", 0, null, null, null, true)
                 ),
                 new FileOutputStream(tmpFile));
+
```
Comment (ahsu1230) [inline winterfell/pdf-filler/src/test/java/gov/va/vetaffi/PDFStampingTest.java]: nit: extra line?

File: winterfell/webapp/src/js/formCtrl.js [hunk 2]
```diff
@@ -5,13 +5,16 @@
 var app = angular.module('vetafiApp');
 app.controller("formCtrl", ['$scope', '$filter', '$rootScope', 'formRenderingService', 'formTemplateService', '$routeParams',
     function ($scope, $filter, $rootScope, formRenderingService, formTemplateService, $routeParams) {
-        $scope.$watch("signature", function (newVal, oldVal) {
+        $scope.$watch('signature', function (newVal, oldVal) {
```
Comment (ahsu1230) [inline winterfell/webapp/src/js/formCtrl.js]: where is this signature variable coming from?

File: winterfell/pdf-filler/src/main/java/gov/va/vetaffi/PDFStamping.java [hunk 3]
```diff
@@ -25,6 +26,52 @@
...
+    }
+
+    private static void stampCheckbox(
+            String key,
+            Boolean value,
+            AcroFields form,
+            PdfStamper stamper,
+            PdfReader reader) throws DocumentException, IOException {
+        if (value) {
+            form.setField(key, "1");
```
Comment (ahsu1230) [inline winterfell/pdf-filler/src/main/java/gov/va/vetaffi/PDFStamping.java]: does this mean checked? Looks a bit hard-coded so a comment would be nice

**Comment by ahsu1230** [comment]
lgtm

File: winterfell/webapp/src/js/formCtrl.js (hunk 2 above)
Comment (qwwqwwq) [inline winterfell/webapp/src/js/formCtrl.js]: from the signature_pad.js directive

File: winterfell/pdf-filler/src/main/java/gov/va/vetaffi/PDFStamping.java (hunk 3 above)
Comment (qwwqwwq) [inline winterfell/pdf-filler/src/main/java/gov/va/vetaffi/PDFStamping.java]: youre right i should make this a constant

**Pull request merged**